lib/bpf/ubpf_hashmap.c

```diff
@@ -60,6 +60,37 @@ static inline struct hmap_elem* lookup_elem_raw(struct ovs_list *head,
     return NULL;
 }
 
+unsigned int
+ubpf_hashmap_dump(const struct ubpf_map *map, void **data)
+{
+    struct hashmap *hmap = map->data;
+    const struct ovs_list *head;
+    struct hmap_elem *element;
+    int key_size = map->key_size;
+    int value_size = map->value_size;
+    int key_rounded_size = round_up(map->key_size, 8);
+
+    *data = malloc(hmap->count * (key_size + value_size));
```

Comment:
I would still prefer to have the caller (i.e., `handle_dump_map` here) do the allocation.